client/frontend/retryableClient.go

```diff
@@ -51,584 +58,409 @@ func NewRetryableClient(
 	}
 }
 
-func (c *retryableClient) DeprecateDomain(
-	ctx context.Context,
-	request *types.DeprecateDomainRequest,
-	opts ...yarpc.CallOption,
-) error {
+func (c *retryableClient) CountWorkflowExecutions(ctx context.Context, cp1 *types.CountWorkflowExecutionsRequest, p1 ...yarpc.CallOption) (cp2 *types.CountWorkflowExecutionsResponse, err error) {
```

Comment:
I kinda wonder if it'd be worth sorting these before regenerating so they mostly line up...

Reply:
sorted and checked by hand: all changes look good / obviously identical, tho those 3 explicit exceptions are a bit of a❓.